addons/sale/models/sale_order_line.py

```diff
@@ -1177,6 +1177,12 @@ def create(self, vals_list):
             if vals.get('display_type') or self.default_get(['display_type']).get('display_type'):
                 vals['product_uom_qty'] = 0.0
 
+            if 'technical_price_unit' in vals and 'price_unit' not in vals:
+                # price_unit field was set as readonly in the view (but technical price unit not)
```

Comment:
Nitty nit (same below):
```suggestion
                # price_unit field was set as readonly in the view (but technical_price_unit not)
```